Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -1495,6 +1495,74 @@ def messages() -> Response:
     return Response(status=200)
 
 
+def ring_user():
+    """Rings a user on Teams.
+
+    Notes:
+        This is a ring only! no media plays in case the generated call is answered.
+
+    Returns:
+        None.
+    """
+    bot_id = demisto.params().get('bot_id')
+    tenant_id = demisto.args().get('tenant_id')
+    callback_uri = demisto.args().get('callback_uri', 'https://callback.url')
```

Comment:
uri or url?
Add better explanation

Reply:
it is called callback URI in the api but I can change to URL if you think it is more understandable